common/rc

```diff
@@ -140,3 +143,101 @@ _get_pci_dev_from_blkdev() {
 	readlink -f "$TEST_DEV_SYSFS/device" | \
 		grep -Eo '[0-9a-f]{4}:[0-9a-f]{2}:[0-9a-f]{2}\.[0-9a-f]'
 }
+
+_notrun()
+{
+	echo "$*" > $seqres.notrun
+	echo "$seq not run: $*"
+	rm -f ${RESULT_DIR}/require_test*
+	rm -f ${RESULT_DIR}/require_scratch*
+
+	status=0
+	exit
+}
+
+_fail()
+{
+	echo "$*" | tee -a $seqres.full
+	echo "(see $seqres.full for details)"
+	status=1
+	exit 1
+}
+
+_require_loop()
+{
+	modprobe loop >/dev/null 2>&1
+	if grep loop /proc/devices >/dev/null 2>&1
+	then
+		:
+	else
+		_notrun "This test requires loopback device support"
+	fi
+}
+
+_create_loop_device()
+{
+	file=$1
+	dev=`losetup -f --show $file` || _fail "Cannot assign $file to a loop device"
+	echo $dev
+}
+
+_destroy_loop_device()
+{
+	dev=$1
+	losetup -D $dev || _fail "Cannot destroy loop device $dev"
+}
+
+_mkfs_dev()
```

Comment:
This is way too much xfstests baggage. If a test really needs this, then it probably belongs in xfstests and not blktests. Otherwise just call mkfs directly.